shared/src/main/kotlin/org/jetbrains/kotlin/konan/target/Linker.kt

```diff
@@ -270,6 +269,51 @@ open class LinuxBasedLinker(targetProperties: LinuxBasedConfigurables)
     }
 }
 
+open class FrcLinker(targetProperties: FrcConfigurables)
```

Comment:
It seems unnecessary to have a separate class for this, but modifying the `LinuxBasedLinker` to also support the FRC target got very messy.